packages/gestalt/src/Icon.js

```diff
@@ -90,6 +90,8 @@ const flipOnRtlIconNames = [
 
 /**
  * [Icons](https://gestalt.pinterest.systems/icon) are the symbolic representation of an action or information, providing visual context and improving usability.
+ *
+ * See the [Iconography and SVG guidelines](/iconography_and_svgs) to explore the full icon library.
```

Comment:
Added this so that it's visible without opening the props table

Reply:
can you use the full URL here so the link works in IDEs as well?